test/parallel/test-crypto-certificate.js

```diff
@@ -30,9 +30,9 @@ const { Certificate } = crypto;
 const fixtures = require('../common/fixtures');
 
 // Test Certificates
-const spkacValid = fixtures.readKey('rsa_spkac.spkac');
+const spkacValid = fixtures.readKey('rsa_spkac.spkac', null, true);
```

Comment:
If taking previous suggestions from me:

```suggestion
const spkacValid = fixtures.readKey('rsa_spkac.spkac', null, { trim: true });
```